include/sdf/Ellipsoid.hh

```diff
@@ -64,6 +67,12 @@ namespace sdf
     /// \return A reference to a gz::math::Ellipsoidd object.
     public: gz::math::Ellipsoidd &Shape();
 
+    /// \brief Calculate and return the Mass Matrix values for the Ellipsoid
+    /// \param[in] density Density of the ellipsoid in kg/m^3
```

Comment:
```suggestion
    /// \param[in] _density Density of the ellipsoid in kg/m^3
```

Reply:
Corrected in [1f16e1f](https://github.com/gazebosim/sdformat/pull/1304/commits/1f16e1f0a5454acc5a30a4b86b8f273b1535eb95)